From 34143055b3291575f472f405cd2cba37f03aaf49 Mon Sep 17 00:00:00 2001
From: Armando Uribe De Leon <x0095078@ti.com>
Date: Mon, 3 May 2010 19:03:52 -0500
Subject: [PATCH 118/135] DSPBRIDGE: Remove checkpatch warnings: 80 char lines

This patch removes checkpatch warning regarding with
lines exceeded 80 chatacters.

Signed-off-by: Armando Uribe De Leon <x0095078@ti.com>
---
 arch/arm/plat-omap/include/dspbridge/dev.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/wmdmsg.h |    3 +-
 drivers/dsp/bridge/pmgr/dev.c                 |    3 +-
 drivers/dsp/bridge/rmgr/node.c                |   71 +++++++++++--------------
 drivers/dsp/bridge/rmgr/pwr.c                 |    4 +-
 drivers/dsp/bridge/wmd/chnl_sm.c              |    4 +-
 6 files changed, 40 insertions(+), 47 deletions(-)

diff --git a/arch/arm/plat-omap/include/dspbridge/dev.h b/arch/arm/plat-omap/include/dspbridge/dev.h
index c9a1a5a..226c9d6 100644
--- a/arch/arm/plat-omap/include/dspbridge/dev.h
+++ b/arch/arm/plat-omap/include/dspbridge/dev.h
@@ -392,7 +392,7 @@ extern struct dev_object *dev_get_first(void);
  *      else:           *ppIntfFxns is NULL.
  */
 extern dsp_status dev_get_intf_fxns(struct dev_object *hdev_obj,
-				    OUT struct bridge_drv_interface **ppIntfFxns);
+			    OUT struct bridge_drv_interface **ppIntfFxns);
 
 /*
  *  ======== dev_get_io_mgr ========
diff --git a/arch/arm/plat-omap/include/dspbridge/wmdmsg.h b/arch/arm/plat-omap/include/dspbridge/wmdmsg.h
index f8005ea..74d6cbb 100644
--- a/arch/arm/plat-omap/include/dspbridge/wmdmsg.h
+++ b/arch/arm/plat-omap/include/dspbridge/wmdmsg.h
@@ -50,6 +50,7 @@ extern dsp_status bridge_msg_register_notify(struct msg_queue *msg_queue_obj,
 					  struct dsp_notification
 					  *hnotification);
 
-extern void bridge_msg_set_queue_id(struct msg_queue *msg_queue_obj, u32 msgq_id);
+extern void bridge_msg_set_queue_id(struct msg_queue *msg_queue_obj,
+					u32 msgq_id);
 
 #endif /* WMDMSG_ */
diff --git a/drivers/dsp/bridge/pmgr/dev.c b/drivers/dsp/bridge/pmgr/dev.c
index 3465b52..b6cb933 100644
--- a/drivers/dsp/bridge/pmgr/dev.c
+++ b/drivers/dsp/bridge/pmgr/dev.c
@@ -65,7 +65,8 @@ struct dev_object {
 	u8 dev_type;		/* Device Type */
 	struct cfg_devnode *dev_node_obj;	/* Platform specific dev id */
 	struct wmd_dev_context *hwmd_context;	/* WMD Context Handle */
-	struct bridge_drv_interface wmd_interface;	/* Function interface to WMD. */
+	/* Function interface to WMD. */
+	struct bridge_drv_interface wmd_interface;
 	struct brd_object *lock_owner;	/* Client with exclusive access. */
 	struct cod_manager *cod_mgr;	/* Code manager handle. */
 	struct chnl_mgr *hchnl_mgr;	/* Channel manager. */
diff --git a/drivers/dsp/bridge/rmgr/node.c b/drivers/dsp/bridge/rmgr/node.c
index d63761f..a889b24 100644
--- a/drivers/dsp/bridge/rmgr/node.c
+++ b/drivers/dsp/bridge/rmgr/node.c
@@ -2444,46 +2444,37 @@ dsp_status node_terminate(struct node_object *hnode, OUT dsp_status *pstatus)
 
 		status = (*intf_fxns->pfn_msg_put) (hnode->msg_queue_obj, &msg,
 						    hnode->utimeout);
-		/* FIXME */
-		if (DSP_SUCCEEDED(status)) {
-			/*  Wait on synchronization object that will be
-			 *  posted in the callback on receiving RMS_EXIT
-			 *  message, or by node_delete. Check for valid hnode,
-			 *  in case posted by node_delete(). */
-			status = sync_wait_on_event(hnode->sync_done,
-						    kill_time_out / 2);
-			if (DSP_FAILED(status)) {
-				if (status == -ETIME) {
-					status = (*intf_fxns->pfn_msg_put)
-					    (hnode->msg_queue_obj, &killmsg,
-					     hnode->utimeout);
-					if (DSP_SUCCEEDED(status)) {
-						status = sync_wait_on_event
-						    (hnode->sync_done,
-						     kill_time_out / 2);
-						if (DSP_FAILED(status)) {
-							/* Here it goes the part
-							 * of the simulation of
-							 * the DSP exception */
-							dev_get_deh_mgr
-							   (hnode_mgr->hdev_obj,
-							    &hdeh_mgr);
-							if (hdeh_mgr) {
-								(*intf_fxns->
-								pfn_deh_notify)
-								 (hdeh_mgr,
-								  DSP_SYSERROR,
-								  DSP_EXCEPTIONABORT);
-								status =
-								    -EPERM;
-							}
-						} else
-							status = DSP_SOK;
-					}
-				} else
-					status = -EPERM;
-			} else	/* Convert SYNC status to DSP status */
-				status = DSP_SOK;
+		if (DSP_FAILED(status))
+			goto func_cont;
+
+		/*
+		 * Wait on synchronization object that will be
+		 * posted in the callback on receiving RMS_EXIT
+		 * message, or by node_delete. Check for valid hnode,
+		 * in case posted by node_delete().
+		 */
+		status = sync_wait_on_event(hnode->sync_done,
+					    kill_time_out / 2);
+		if (status != ETIME)
+			goto func_cont;
+
+		status = (*intf_fxns->pfn_msg_put)(hnode->msg_queue_obj,
+						&killmsg, hnode->utimeout);
+		if (DSP_FAILED(status))
+			goto func_cont;
+		status = sync_wait_on_event(hnode->sync_done,
+					     kill_time_out / 2);
+		if (DSP_FAILED(status)) {
+			/*
+			 * Here it goes the part of the simulation of
+			 * the DSP exception.
+			 */
+			dev_get_deh_mgr(hnode_mgr->hdev_obj, &hdeh_mgr);
+			if (!hdeh_mgr)
+				goto func_cont;
+
+			(*intf_fxns->pfn_deh_notify)(hdeh_mgr, DSP_SYSERROR,
+							DSP_EXCEPTIONABORT);
 		}
 	}
 func_cont:
diff --git a/drivers/dsp/bridge/rmgr/pwr.c b/drivers/dsp/bridge/rmgr/pwr.c
index 49b1894..4262109 100644
--- a/drivers/dsp/bridge/rmgr/pwr.c
+++ b/drivers/dsp/bridge/rmgr/pwr.c
@@ -55,8 +55,8 @@ dsp_status pwr_sleep_dsp(IN CONST u32 sleepCode, IN CONST u32 timeout)
 			continue;
 		}
 		if (DSP_FAILED(dev_get_intf_fxns(hdev_obj,
-						 (struct bridge_drv_interface **)
-						 &intf_fxns))) {
+						(struct bridge_drv_interface **)
+						&intf_fxns))) {
 			continue;
 		}
 		if (sleepCode == PWR_DEEPSLEEP)
diff --git a/drivers/dsp/bridge/wmd/chnl_sm.c b/drivers/dsp/bridge/wmd/chnl_sm.c
index 67bd0df..d62acf2 100644
--- a/drivers/dsp/bridge/wmd/chnl_sm.c
+++ b/drivers/dsp/bridge/wmd/chnl_sm.c
@@ -511,8 +511,8 @@ dsp_status bridge_chnl_flush_io(struct chnl_object *chnl_obj, u32 dwTimeOut)
 			 * timeout: */
 			while (!LST_IS_EMPTY(pchnl->pio_requests) &&
 			       DSP_SUCCEEDED(status)) {
-				status = bridge_chnl_get_ioc(chnl_obj, dwTimeOut,
-							  &chnl_ioc_obj);
+				status = bridge_chnl_get_ioc(chnl_obj,
+						dwTimeOut, &chnl_ioc_obj);
 				if (DSP_FAILED(status))
 					continue;
 
-- 
1.7.0.4

